src/app/chat/signal_handling.nim

```diff
@@ -1,7 +1,7 @@
 proc handleSignals(self: ChatController) =
   self.status.events.on(SignalType.Message.event) do(e:Args):
     var data = MessageSignal(e)
-    self.status.chat.update(data.chats, data.messages, data.emojiReactions, data.communities)
+    self.status.chat.update(data.chats, data.messages, data.emojiReactions, data.communities, data.membershipRequests)
```

Comment:
Not on this PR, but do you think we should change this proc to receive the message signal instead of adding a new parameter for any new item the message signal includes in the future?